crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -176,6 +176,27 @@ def create_proposal
     base
   end
 
+  def disable_non_core_proposals
```

Comment:
I am not completely sure if CrowbarService is really the right place for this. I am interested in feedback.